config/G18_02a/TuneGeneratorList.xml

```diff
@@ -47,15 +47,15 @@ Generator-%d                alg     No
 -->
 
   <param_set name="Default"> 
-     <param type="int" name="NGenerators">   18                                    </param>
+     <param type="int" name="NGenerators">   19                                    </param>
```

Comment:
That's THE question. I would advocate for a yes in this specific case. I have a few reasons for it: 
 - it's process we know should be there, we just didn't have a model so far
 - It's an NC process so this will not impact the present tunes that only took into account CC interaction
 - There is desire from experiments already
 Having said that, this is something we should discuss at the review. That is why I only changed one, to be used for testing and developing.